chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -64,7 +64,7 @@
 # the server.  In most cases, the default value should be fine.
 #
 # tunnel_bridge = br-tun
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") || @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
There's not much to choose between the two versions: use of `&` and `any?` is ever so slightly cryptic, but OTOH repetition of `@ml2_type_drivers.include?` is ugly and requires extra effort for the brain and eyes to check that it really is exact repetition.  What swings it for me is the scalability of my version: if other cases need to be added later, then the temptation to copy'n'paste resulting in a 3rd `@ml2_type_drivers.include?()` call will be too great to resist :)